pages/common/histexpand.md

```diff
@@ -19,13 +19,17 @@
 
 `{{command}} !^`
 
+- Run the Nth command of the history:
+
+`!{{N}}`
+
 - Run the command `n` lines back in the history:
 
 `!-{{n}}`
 
 - Run the most recent command starting with `string`:
```

Comment:
This line should probably update too.  How about
```suggestion
- Run the most recent command containg with `string`:
```

Reply:
`containing` @bl-ue ?

Reply:
yeah, @bl-ue is right

man history:
_!?string[?] Refer to the most recent command preceding the current position in the history list containing string.  The trailing ? may be omitted if string is followed immediately by a newline._